### DISCUSSION TITLE
[HLS] ScaleHLS: Achieving Scalable High-Level Synthesis through MLIR

### DISCUSSION THREAD

**Discussion by hanchenye** [cucapra/latte21#37] [Papers]
Authors: Hanchen Ye, Cong Hao, Hyunmin Jeong, Jack Huang, and Deming Chen
Paper PDF: https://hanchenye.com/file/LATTE21_ScaleHLS_Hanchen.pdf
Talk Recording: https://youtu.be/BkE-3upeHUE

**Comment by rachitnigam** [comment]
@hanchenye Your YouTube talk is marked as "content for kids" which means we can't add it to our playlist. Can you update the setting for the video and mark it not for kids? Please ping me (by saying @rachitnigam) when you're done.

**Reply by hanchenye** [reply]
Hey @rachitnigam I've updated the video setting. Please have a try.

**Reply by rachitnigam** [reply]
Worked! Thanks.
